doc/api/async_hooks.md

```diff
@@ -203,13 +203,16 @@ UDPSENDWRAP, UDPWRAP, WRITEWRAP, ZLIB, SSLCONNECTION, PBKDF2REQUEST,
 RANDOMBYTESREQUEST, TLSWRAP
 ```
 
+There is also the `PROMISE` resource type, which is used to track `Promise`
```

Comment:
Hm. Seems we don't even include `Timer` or `TickObject` in that list. Or even as a side comment.

Anyway, only argument I'd have for adding it to the above list is because it shows up in `process.binding('async_wrap').Providers`. But that's not a strong argument. I'll leave this up to you.